src/main/java/ch/njol/skript/expressions/ExprBannerItem.java

```diff
@@ -0,0 +1,101 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import org.bukkit.Material;
+import org.bukkit.block.banner.PatternType;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+@Name("Banner Pattern Item")
+@Description({
+	"Gets the item from a banner pattern type.",
+	"NOTE: Not all banner patterns have an item. Currently there are:",
+	"<ul>",
+	"<li>creeper/creeper charged</li>",
+	"<li>bricks/field masoned</li>",
+	"<li>piglin/snout</li>",
+	"<li>mojang/thing</li>",
+	"<li>border/bordure indented</li>",
+	"<li>flow, flower, globe, guster, skull</li>",
+	"</ul>"
```

Comment:
why is this a bullet pointed list, with multiple for each bullet point? 
The list shouldn't even be here, it should be on the type's documentation.